pkg/controller/operators/catalog/operator_test.go

```diff
@@ -1119,6 +1167,15 @@ func objectReference(name string) *corev1.ObjectReference {
 	return &corev1.ObjectReference{Name: name}
 }
 
+func yamlFromFilePath(fileName string) string {
```

Comment:
I understand what you are doing here but I wonder if that file doesn't exist and the func returns empty string, would the test would pass or fail or what would be expected result if that is the case. Maybe I'm bit over cautious here but if there is an error, we should return and check for it, simply avoid some false positive in case the test file being moved.

Reply:
Adding an error check.